docs/admin/setup/pyris.rst

```diff
@@ -1,100 +1,332 @@
+       - ``json_mode``: Whether the model supports structured JSON output mode.
+       - ``privacy_compliance``: Whether the model complies with privacy regulations.
+       - ``self_hosted``: Whether the model is self-hosted.
+       - ``vendor``: The provider of the model (e.g., OpenAI).
+       - ``speed``: The model's processing speed.
+
+     - ``description``: Additional information about the model.
+     - ``id``: Unique identifier for the model across all models.
+     - ``model``: The official name of the model as used by the vendor.
+     - ``name``: A custom, human-readable name for the model.
+     - ``type``: The model type, used to select the appropriate client (e.g., ``openai_chat``, ``azure_chat``, ``ollama``).
+     - ``endpoint``: The URL to connect to the model.
+     - ``api_version``: The API version to use with the model.
+     - ``azure_deployment``: The deployment name of the model on Azure.
+     - ``tools``: The tools supported by the model. For now, we do not provide any predefined tools, but the field is necessary for the models with tool calling capabilities.
+
+     **Notes on ``gpt_version_equivalent``:**
+
+     The ``gpt_version_equivalent`` field is subjective and used to compare capabilities of different models using GPT models as a reference. For example:
+
+     - GPT-4 Omni equivalent: 4.5
+     - GPT-4 Omni Mini equivalent: 4.25
+
+     .. warning::
+
+        Most existing pipelines in Pyris require a model with a ``gpt_version_equivalent`` of 4.5 or higher. It is advised to define models in the ``llm_config.local.yml`` file with a ``gpt_version_equivalent`` of 4.5 or higher.
+
+4. **Run the Server**
+
+   Start the Pyris server:
+
+   .. code-block:: bash
+
+      APPLICATION_YML_PATH=./application.local.yml \
+      LLM_CONFIG_PATH=./llm_config.local.yml \
+      uvicorn app.main:app --reload
+
+5. **Access API Documentation**
+
+   Open your browser and navigate to `http://localhost:8000/docs` to access the interactive API documentation.
+
+This setup should help you run the Pyris application on your local machine. Ensure you modify the configuration files as per your specific requirements before deploying.
+
+Using Docker
+------------
+
+**Prerequisites**
+
+- Ensure Docker and Docker Compose are installed on your machine.
+- Clone the Pyris repository to your local machine.
+- Create the necessary configuration files as described in the previous section.
+
+**Docker Compose Files**
+
+- **Development**: ``docker-compose/pyris-dev.yml``
+- **Production with Nginx**: ``docker-compose/pyris-production.yml``
+- **Production without Nginx**: ``docker-compose/pyris-production-internal.yml``
+
+**Setup Instructions**
+
+1. **Running the Containers**
+
+   You can run Pyris in different environments: development or production.
+
+   **Development Environment**
+
+   - **Start the Containers**
+
+     .. code-block:: bash
+
+        docker-compose -f docker-compose/pyris-dev.yml up --build
```

Comment:
The docker-compose folder also needs to be changed here to docker/

Reply:
Thanks for noticing